examples/llama/model_test.py

```diff
@@ -155,19 +162,22 @@ def test_module(
             for k, v in state_dict_pt.items()
         }
     else:
-        raise ValueError(f"Cannot find the given path: {pth_path}")
+        raise ValueError(f"Cannot find the given path: {ckpt_dir}")
 
     # Run the ARK module
     res_ark = run_ark(
         module_ark,
         state_dict_ark,
         inputs_ark,
         iterations=test_thru_iterations if test_thru else 1,
+        rank=rank,
+        world_size=world_size,
     )
 
     if not test_thru_ark_only:
         # PyTorch module
         module_pt: torch.nn.Module = module_class_pt(*module_args_pt)
+        module_pt.to(dtype=numpy_dtype_to_torch_dtype[dtype])
```

Comment:
Can you try [`torch.set_default_dtype`](https://pytorch.org/docs/stable/generated/torch.set_default_dtype.html#torch-set-default-dtype) instead? I think this is the official way to do this.